cont/LuaUI/rml_setup.lua

```diff
@@ -0,0 +1,27 @@
+--  file:    rml.lua
+--  brief:   RmlUi Setup
+--  author:  lov + ChrisFloofyKitsune
+--
+--  Copyright (C) 2024.
+--  Licensed under the terms of the GNU GPL, v2 or later.
+
+if (RmlGuard or not RmlUi) then
+	return
+end
+RmlGuard = true
+
+-- Load fonts
+RmlUi.LoadFontFace("Fonts/FreeMonoBold.ttf", true)
+
+-- Mouse Cursor Aliases
+--[[
+	These let standard CSS cursor names be used when doing styling.
+	If a cursor set via RCSS does not have an alias, it is unchanged.
+	CSS cursor list: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor
+	RmlUi documentation: https://mikke89.github.io/RmlUiDoc/pages/rcss/user_interface.html#cursor
+]]
+
+-- when "cursor: normal" is set via RCSS, "cursornormal" will be sent to the engine
+RmlUi.SetMouseCursorAlias("default", 'cursornormal')
```

Comment:
what happens if a game doesn't do this?

Reply:
Then the .rcss files will have to specify the game specific cursor name to use

instead of
```css
.example {
    cursor: default
}
```

they would have to put
```css
.example {
    cursor: cursornormal
}
```

the latter of which makes css parsers confused and css code in general less importable if it was developed elsewhere

Reply:
also comment says "cursor: normal", shouldn't it be "cursor: default"?

Reply:
> instead of
> 
> ```css
> .example {
>     cursor: default
> }
> ```
> 
> they would have to put
> 
> ```css
> .example {
>     cursor: cursornormal
> }
> ```

what if they put `default` there regardless?

Or, say my widget wants the "wait" cursor (hourglass). since this file doesn't set it, will that fail?

Reply:
same as if any other part of the code that tries to use a cursor that's not registered with the engine-
it will default to "" in the MouseHandler code

https://github.com/beyond-all-reason/spring/blob/fc1bb20fa2e41c0aacdc5dd5dfae8a8fe0aafc8d/rts/Game/UI/MouseHandler.cpp#L835-L859

which is assigned here in the init/reload code
https://github.com/beyond-all-reason/spring/blob/fc1bb20fa2e41c0aacdc5dd5dfae8a8fe0aafc8d/rts/Game/UI/MouseHandler.cpp#L160-L163

Reply:
if every command defaults to `""` if missing, and `""` command is assigned the `"cursornormal"` cursors in engine, then does `RmlUi.SetMouseCursorAlias("default", 'cursornormal')` actually do anything?

a rml widget will say `.example { cursor: default }`, this will fallback to `""` because `"default"` is missing, and `""` uses `"cursornormal"` anyway, no?

Reply:
What is the worst case outcome of this? If the worst case outcome is fairly minor, it would be fine to improve this, but not something that should block the PR, in my perspective at least.

Reply:
My aim is to
* see if it would be possible to get rid of this file
* become aware of the caveats etc for documentation purposes if not

Reply:
the version of this file I have in my rmlui-example-widgets branch has a lot more going on
I stripped down that file to make this one

https://github.com/ChrisFloofyKitsune/Beyond-All-Reason/blob/rmlui-example-widgets/luaui/rml_setup.lua

in terms of useful stuff, 4 fonts are loaded and 5 cursor aliases are set there